modules/rosa-hcp-sharing-vpc-creation-and-sharing.adoc

```diff
@@ -0,0 +1,364 @@
+
+image::522-shared-vpc-step-1.png[]
+.Procedure
+
+. Create or modify a VPC to your specifications in the link:https://us-east-1.console.aws.amazon.com/vpc/[VPC section of the AWS console].
++
+. Create the `Route 53 role`.
++
+[NOTE]
+====
+You must create the `Route 53 role` in the same account as the hosted zones (which are created in Step 3). For example, if you want to create the hosted zones in the centrally-managed VPC account, you must create the `Route 53 role` in the *VPC Owner* account. If you want to create the hosted zones in the workload account, you must create the `Route 53 role` in the *Cluster Creator* account.
+====
++
+.. Create a custom policy file to allow for necessary shared VPC permissions that uses the name `Route53Policy`:
++
+[source,terminal]
+----
+$ cat <<EOF > /tmp/route53-policy.json
+{
+  "Version" : "2012-10-17",
+  "Statement" : [
+    {
+      "Sid" : "ReadPermissions",
+      "Effect" : "Allow",
+      "Action" : [
+        "elasticloadbalancing:DescribeLoadBalancers",
+        "route53:GetHostedZone",
+        "route53:ListResourceRecordSets",
+	"route53:ListHostedZones",
+        "tag:GetResources"
+      ],
+      "Resource" : "*"
+    },
+    {
+      "Sid" : "ChangeResourceRecordSetsRestrictedRecordNames",
+      "Effect" : "Allow",
+      "Action" : [
+        "route53:ChangeResourceRecordSets"
+      ],
+      "Resource" : [
+        "*"
+      ],
+      "Condition" : {
+        "ForAllValues:StringLike" : {
+          "route53:ChangeResourceRecordSetsNormalizedRecordNames" : [
+            "*.hypershift.local",
+            "*.openshiftapps.com",
+            "*.devshift.org",
+            "*.openshiftusgov.com",
+            "*.devshiftusgov.com"
+          ]
+        }
+      }
+    },
+    {
+      "Sid" : "ChangeTagsForResourceNoCondition",
+      "Effect" : "Allow",
+      "Action" : [
+        "route53:ChangeTagsForResource"
+      ],
+      "Resource" : "*"
+    }
+  ]
+}
+----
++
+.. Create the policy in AWS:
++
+[source,terminal]
+----
+$ aws iam create-policy \
+    --policy-name Route53Policy \
+    --policy-document file:///tmp/route53-policy.json
+----
++
+You will attach this policy to a role necessary for the shared VPC permissions.
++
+.. Create a custom trust policy file that grants permission to assume roles:
++
+[source,terminal]
+----
+$ cat <<EOF > /tmp/route53-role.json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::<Account-ID>:root"  <1>
+            },
+            "Action": "sts:AssumeRole"
+        }
+    ]
+}
+EOF
+----
++
+--
+<1> The principal is reduced in scope after the *Cluster Creator* creates the necessary cluster roles. On creation, you must create a root user placeholder by using the *Cluster Creator's* AWS account ID as `arn:aws:iam::{Account}:root`.
```

Comment:
During the initial creation of the principal, we should use the *VPC Owner* AWS account ID instead of the *Cluster Creator* AWS account ID. This is only a temporary placeholder and we should better explain that this is simply a placeholder.